models/news.model.js

```diff
@@ -166,6 +167,12 @@ const deleteCommentById = (id) => {
     });
 };
 
+const fetchEndpoints = () => {
```

Comment:
This should be happening in the controller. We're not interacting with the database here, so just move this logic across to your controller file. Are there also any optional parameters to readFile that could format the string in a more readable format?